packages/dx-scheduler-core/src/plugins/integrated-grouping/helpers.ts

```diff
@@ -32,3 +35,103 @@ export const addGroupInfoToCells: PureComputed<
   }
   return { ...viewCell, groupingInfo, hasRightBorder: true };
 });
+
+const getCurrentGroup: PureComputed<
+  [GroupingItem[][], ValidResource[], number, GroupingItem], GroupingItem[]
+> = (groupingItems, resources, index, groupingItem) => {
+  let currentIndex = index;
+  return groupingItems.reduceRight((groupAcc, groupingItemsRow, rowIndex) => {
+    if (rowIndex === groupingItems!.length - 1) return groupAcc;
```

Comment:
We prefer to use curly braces in early return so that it don't get lost in a long string:
```
if (rowIndex === groupingItems!.length - 1) {
  return groupAcc;
}
```